### PR TITLE
Card870 - Add message for users permissions

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/view/components/rule-engine/rule-engine.container.ts [hunk 1]
```diff
@@ -91,6 +93,8 @@ export class RuleEngineContainer {
   rules$:EventEmitter<RuleModel[]> = new EventEmitter()
   ruleActions$:EventEmitter<ActionModel[]> = new EventEmitter()
   conditionGroups$:EventEmitter<ConditionGroupModel[]> = new EventEmitter()
+  globalError:string;
+  showRules:boolean
```
Comment (ggranum) [inline src/view/components/rule-engine/rule-engine.container.ts]: Are you actually using this field? It looks like you're setting the value on the 'state' object, not 'this'.

**Comment by ggranum** [comment]
I think there the two field variables on rule-engine.container.ts aren't used, but otherwise it's awesome.

**Comment by jtesser** [comment]
Thanx @ggranum we are updating the pull now.

File: src/view/components/rule-engine/rule-engine.container.ts (hunk 1 above)
Comment (fmontes) [inline src/view/components/rule-engine/rule-engine.container.ts]: You are right, fix sent

**Pull request merged**